packages/cli/src/executable/AgenticaStart.ts

```diff
@@ -34,53 +41,102 @@ export namespace AgenticaStart {
       a.name.localeCompare(b.name),
     );
 
-    const questions = getQuestions({ services: availableServices });
+    const questions = getQuestions({ services: availableServices }, options);
     const answers = await inquirer.prompt(questions);
 
+    const updatedAnswers = {
+      ...answers,
+      // options
+      projectType: options.project ?? answers.projectType,
+    };
+
     const validAnswers = typia.assert<{
-      packageManager: "npm" | "yarn" | "pnpm";
+      packageManager: PackageManager;
       openAIKey: string;
-      // projectType: string;
       services: string[];
-    }>(answers);
+      // Options
+      projectType: ProjectOptionValue;
+    }>(updatedAnswers);
 
-    const {
-      packageManager,
-      openAIKey,
-      // projectType,
-      services,
-    }: {
-      packageManager: "npm" | "yarn" | "pnpm";
-      openAIKey: string;
-      services: string[];
-    } = validAnswers;
+    const { packageManager, openAIKey, projectType, services } = validAnswers;
 
-    // Create project folder
-    const { projectPath } = createProjectDirectory({ projectName });
+    if (projectType === "standalone") {
```

Comment:
The detailed logic for branching does not seem to fall under the responsibility of this function.
Would it be possible to separate the function?

If an if statement becomes too long, it can cause excessive cognitive load for the reader.